data/autoyast_sle15/autoyast_media_up_live_gnome.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-15-SP1-Module-Basesystem-TEST/x86_64/DVD1/]]></media_url>
```

Comment:
what would happen when SLE 15 SP1 will be released ? as far as I know this is right way to go for this issue : 
```suggestion
        <media_url>{{BASE_MODULE_URL}}</media_url>
```

@rwx788 WDYT ?

Reply:
+1 That's preferable, but we need code adjustments to expand this one.

Reply:
Yes, I agree this improvement. I will try to modify code to realize it.